google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/AppProfile.java

```diff
@@ -409,4 +412,108 @@ public int hashCode() {
       return Objects.hashCode(proto);
     }
   }
+
+  /** Compute Billing Owner specifies how usage should be accounted when using Data Boost. */
+  public static enum ComputeBillingOwner {
+    UNSPECIFIED(DataBoostIsolationReadOnly.ComputeBillingOwner.COMPUTE_BILLING_OWNER_UNSPECIFIED),
+    HOST_PAYS(DataBoostIsolationReadOnly.ComputeBillingOwner.HOST_PAYS);
+
+    private final com.google.bigtable.admin.v2.AppProfile.DataBoostIsolationReadOnly
+            .ComputeBillingOwner
+        proto;
+
+    /**
+     * Wraps the protobuf. This method is considered an internal implementation detail and not meant
+     * to be used by applications.
+     */
+    @InternalApi
+    public static ComputeBillingOwner fromProto(
+        com.google.bigtable.admin.v2.AppProfile.DataBoostIsolationReadOnly.ComputeBillingOwner
+            proto) {
+      Preconditions.checkNotNull(proto);
+
+      for (ComputeBillingOwner owner : values()) {
+        if (owner.proto.equals(proto)) {
+          return owner;
+        }
+      }
+
+      throw new IllegalArgumentException("Unknown billing owner: " + proto);
```

Comment:
I dont think this should throw, otherwise adding a new variant will cause existing code to break